js/views/storyeditview/storyeditview.js

```diff
@@ -260,16 +260,16 @@ var StoryEditView = Marionette.CompositeView.extend(
     this.addPassage(name, left, top, 'endLevel');
   },
 
-  addPassageEndGameGroup: function(name, left, top) {
-    this.addPassage(name, left, top, 'endGameGroup`');
+  addPassageEndGameGroupSuccessNumberResult: function(name, left, top) {
+    this.addPassage(name, left, top, 'addPassageEndGameGroupSuccessNumberResult');

```

Comment:
The 4th parameter defines the type of passage that should be created. So a type name like `addPassage*`  doesn't really make sense. I'll eventually be making the changes to 'ds' and 'storyConfig' in the lines above, but what would probably be better here is to use: `PassageEndGameGroupSuccessNumberResult.prototype.defaults.type`